servers/src/main/java/tachyon/master/file/meta/PersistenceState.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.file.meta;
+
+/**
+ * The persistence state of a file in the under-storage system.
+ */
+public enum PersistenceState {
+  NOT_PERSISTED, // file not persisted in the under FS
+  SCHEDULED, // used for async persistence, the file is scheduled for persistence on the master,
```

Comment:
The only method that checks if the state is SCHEDULED does `SCHEDULED || PERSISTING`, which suggests we don't need to distinguish between the two. Why do we need both? Could we just replace both of them with "IN_PROGRESS"?